### PR TITLE
Add `protocol` method for Windows.

### PR DESCRIPTION
**Pull request by Berrysoft**
Don't know why this method didn't appear on Windows before:)

### CODE REVIEW

File: src/sys/windows.rs [hunk 1]
```diff
@@ -920,6 +920,21 @@ impl crate::Socket {
             Ok(())
         }
     }
+
+    /// Returns the [`Protocol`] of this socket by checking the `SO_PROTOCOL_INFOW`
+    /// option on this socket.
+    #[cfg(feature = "all")]
+    pub fn protocol(&self) -> io::Result<Option<crate::Protocol>> {
+        use windows_sys::Win32::Networking::WinSock::SO_PROTOCOL_INFOW;
```
Comment (Thomasdezeeuw) [inline src/sys/windows.rs:928]: Nit: can you move this to the other imports.

**Review by Thomasdezeeuw — APPROVED** [review]

File: src/sys/windows.rs (hunk 1 above)
Comment (Berrysoft) [reply src/sys/windows.rs]: Moved.

**Comment by Thomasdezeeuw** [comment]
Thanks @Berrysoft

**Comment by Berrysoft** [comment]
By the way, when would be the next release? @Thomasdezeeuw

**Comment by Thomasdezeeuw** [comment]
> By the way, when would be the next release? @Thomasdezeeuw

Socket2 doesn't have a release schedule. If someone asks I just prepare a release. I can do that this week(end).

**Comment by Berrysoft** [comment]
Thank you! I would like to see a new release.

**Pull request merged**